action.yml

```diff
@@ -345,7 +347,6 @@ runs:
         CERTORA_USE_HARD_LINKS: "${{ inputs.use-hard-links }}"
 
     - name: Add GH Status
-      if: always()
```

Comment:
We were using `always()` so that we receive a comment if any of the previous steps fail. We need to execute it if an exception occurs during the submission step, as this will add a comment with errors and run logs. We can replace it with `success() || failure()` (and maybe some logic for skipped jobs or when we did not execute any jobs).

 The same applies to the steps below.